front/app/containers/Authentication/steps/EmailAndPassword/index.tsx

```diff
@@ -183,7 +183,7 @@ const EmailAndPassword = ({
             {formatMessage(sharedMessages.forgotPassword)}
           </TextLink>
         </Box>
-        <Box mt="12px">
+        <Text mt="12px">
```

Comment:
Can we make a reusable component here to ensure it stays synced across different files?

Reply:
That's a very good question. I don't know. I also raised a similar one here https://citizenlabco.slack.com/archives/CQAGMHKT3/p1685539036962409

It seems in general we prefer to duplicate FE code quite often. In my opinion, we should try to avoid it more often, but this decision should be made on a higher level than this PR. This problem was raised a few times, and the decision was that duplication is, in general, better than a new abstraction.

In this specific case, this duplication wasn't introduced by me, so I didn't change it.